.github/workflows/graalvm-dev.yml

```diff
@@ -0,0 +1,43 @@
+# WARNING: Do not edit this file directly. Instead, go to:
+#
+# https://github.com/micronaut-projects/micronaut-project-template/tree/master/.github/workflows
+#
+# and edit them there. Note that it will be sync'ed to all the Micronaut repos
+name: GraalVM Dev CI
+on:
+  schedule:
+    - cron: "0 1 * * 1-5" # Mon-Fri at 1am UTC
+jobs:
+  build:
+    if: github.repository != 'micronaut-projects/micronaut-project-template'
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        graalvm: [ 'dev' ]
+        java: [ '17' ]
+    steps:
+      - uses: actions/checkout@v3
+      - name: Pre-Build Steps
+        uses: micronaut-projects/github-actions/graalvm/pre-build@master
+        id: pre-build
+        with:
+          graalvm: ${{ matrix.graalvm }}
+          java: ${{ matrix.java }}
+        env:
+          GRADLE_ENTERPRISE_ACCESS_KEY: ${{ secrets.GRADLE_ENTERPRISE_ACCESS_KEY }}
+          GRADLE_ENTERPRISE_CACHE_USERNAME: ${{ secrets.GRADLE_ENTERPRISE_CACHE_USERNAME }}
+          GRADLE_ENTERPRISE_CACHE_PASSWORD: ${{ secrets.GRADLE_ENTERPRISE_CACHE_PASSWORD }}
```

Comment:
Please define all the common env variables at the job level

Reply:
Done

Reply:
Just common or all?

Reply:
Only common, to avoid repeating them in multiple steps

Reply:
Done